scripts/utils/import-metrics-loop.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
```

Comment:
```suggestion

set -o errexit   # abort on nonzero exitstatus
set -o nounset   # abort on unbound variable
set -o pipefail  # don't hide errors within pipes
```

General recommendation for all bash scripts, see https://bertvv.github.io/cheat-sheets/Bash.html

Reply:
Added! Thank you!